Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs

```diff
@@ -141,8 +141,8 @@ public override void OnNetworkSpawn()
         if (IsServer)
         {
             LatestShipColor.Value = m_ShipGlowDefaultColor;
-
-            PlayerName.Value = $"Player {OwnerClientId}";
+            var playerNameString = PlayerName.Value.ToString();
+            playerNameString = $"Player {OwnerClientId}";
```

Comment:
What was this change meant to do?
PlayerName.Value isn't updated anymore so the player names for the ships are now missing.

Reply:
The trigger for this change and some others in the same script was that I had to enter the ""  for `FixedString32Bytes("")` as with the update of NGO it showed me multiple errors in this script regarding strings which I tried to erase. 

I am not sure if I adjusted the strings in the right way and it sounds like I did not. Do you have an idea how the strings in this script should be adjusted instead?

One issue was that it is not possible to set `FixedString32Bytes` to a different var which is a string.

Reply:
Would you be able to share the errors you were encountering?

I reverted both these two lines and the variable's declaration at line 70, and wasn't encountering any errors with Netcode v1.8.1.

Curious to see what you experienced!

Reply:
Wow I feel mocked by my laptop this week... I reverted the changes now again and now it does not show the errors anymore... amazing. But I think with that the problem is solved now. Please let me know if the rest is fine like that and if it can be merged now.